source/_integrations/actronair.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: Actron Air
+description: Instructions on how to integrate the Actron Air A/C controller into Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2025.08
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - "@kclif9"
+ha_domain: actronair
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The **Actron Air** {% term integration %} allows you to control [Actron Air](https://www.actronair.com.au/) Air Conditioning controllers into Home Assistant.
+
+## Prerequisites
+
+You must have an Actron Air Air-Conditioner with the Neo tablet wall controller, an active internet connection, and registered to an email address.
+
+This integration currently supports the Actron Air Neo controller, however other units are intended to be supported soon.
```

Comment:
```suggestion
```
I would move this outside the Prereqs section. I think it deserves a section of its own.